tests/e2e/utils.js

```diff
@@ -9,6 +9,9 @@ export const phrasesList = {
     networkConfigURL: 'https://emerynet.agoric.net/network-config',
     gov1Phrase: Cypress.env('GOV1_PHRASE'),
     gov2Phrase: Cypress.env('GOV2_PHRASE'),
+    // UNTIL https://github.com/Agoric/dapp-econ-gov/issues/144
```

Comment:
what does this comment mean? i dont think that we will be removing these lines after this issue is resolved

Reply:
We won’t remove these lines. Instead, we’ll reverse the addresses. The "UNTIL" comment serves as a reminder and marker for where the update should occur.

Reply:
oh. missed that. thanks for clarifying